mitosheet/mitosheet/code_chunks/replace_code_chunk.py

```diff
@@ -0,0 +1,62 @@
+
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+from typing import List, Tuple
+from mitosheet.code_chunks.code_chunk import CodeChunk
+from mitosheet.types import ColumnID
+from mitosheet.state import State
+import pandas as pd
+
+from mitosheet.errors import MitoError
+
+class ReplaceCodeChunk(CodeChunk):
+
+    def __init__(self, prev_state: State, post_state: State, sheet_index: int, search_value: str, replace_value: str):
+        super().__init__(prev_state, post_state)
+        self.sheet_index = sheet_index
+        self.search_value = search_value
+        self.replace_value = replace_value
+
+    def get_display_name(self) -> str:
+        return 'Replace'
+    
+    def get_description_comment(self) -> str:
+        return f"Replace {self.search_value} with {self.replace_value} in {self.post_state.df_names[self.sheet_index]}"
+
+    def get_code(self) -> Tuple[List[str], List[str]]:
+        search_value = self.search_value
+        replace_value = self.replace_value
+        df = self.post_state.dfs[self.sheet_index]
+        df_name = self.post_state.df_names[self.sheet_index]
```

Comment:
Can you save these in the __init__? Also only reference prev_state, which is another pattern we're trying to move to.